conf/grub.cfg.kvm

```diff
@@ -0,0 +1,5 @@
+# You can put your GRUB overrides here
+set_global load_hv_cmd echo
+set_global load_dom0_cmd linux
+set_global dom0_console console=tty0
+set_global load_initrd_cmd initrd
```

Comment:
One would need to rename this file to /config/grub.cfg for it to come into effect, right?

Would make sense to put that in eve/docs somewhere.

Reply:
Absolutely @eriknordmark -- let me update the README